src/main/java/anita/Database.java

```diff
@@ -0,0 +1,84 @@
+package anita;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Database {
+    private File myFile;
+    private Duke duke;
```

Comment:
Better to change the class name and variable name to `Anita` and `anita` respectively.